code/controllers/subsystem/jobs.dm

```diff
@@ -39,7 +39,7 @@ SUBSYSTEM_DEF(jobs)
 		var/datum/job/job = new J()
 		if(!job)
 			continue
-		if(!job.faction in faction)
+		if((!job.faction) in faction)
```

Comment:
This does absolutely nothing currently and after the change.
`job.faction` is a list. As is `faction` so you'll need to loop through the job.faction list to see if it is in faction. Or the other way around. Depending on the design

Reply:
It does something atleast because it broke the entirety of jobcode when I got the brackets the wrong way round.

Reply:
You checked if a list wasn't in a list with `(!(job.faction) in faction)`. Which would say nope it's not in there since faction is a list of strings. Not a list of lists of strings.
If returning FALSE in the case of `!job.faction in faction` is already something weird... but hey

Reply:
This statement was painful and full of sadness. It is gone now